### ISSUE TITLE
Allow "Attach this PR to bug" link to be opened in any container

### ISSUE THREAD

**Issue by marco-c** [mozilla/rob-bugson#37]
I have Bugzilla open in a container, GitHub open in another container.

The "Attach this PR to bug" link is opening a new tab in the same container as GitHub, which means it opens Bugzilla in the wrong container (that is, a container where I'm not logged in).

Would it be possible to turn it into a real URL so I can right click and open it in a container of my choice?

**Comment by willkg** [comment]
I don't think we can turn it into a real url. It runs javascript to open the page and fill in the appropriate information in the attach form. When I wrote rob-bugson, there was no way to provide that information via querystring params. I don't think that's changed.

It might be possible to completely rewrite rob-bugson to require a bugzilla token and then use the bugzilla api to attach the pr and create merge commits and such. If rob-bugson were structured in such a way, I think it wouldn't need to open pages at all.
